brevbaker-dsl/src/main/kotlin/no/nav/pensjon/brev/template/IncludeAttachment.kt

```diff
@@ -15,6 +16,16 @@ fun <Lang : LanguageSupport, LetterData : Any> createAttachment(
     includeSakspart
 )
 
+fun <Lang : LanguageSupport, LetterData : Any> createAttachment(
+    title: PlainTextOnlyScope<Lang, LetterData>.() -> Unit,
+    includeSakspart: Boolean = false,
+    outline: OutlineOnlyScope<Lang, LetterData>.() -> Unit
+) = AttachmentTemplate<Lang, LetterData>(
+    PlainTextOnlyScope<Lang, LetterData>().apply(title).elements.first(),
```

Comment:
Liker ikke helt first her. Da vil den vel fjerne alle andre elementer? Kan godt tenkes at man vil ha noen showif's osv. Vil helst mer eller mindre kopiere oppsettet brukt i outline, og heller konkatinere tittelen for bruk i vedleggs-lista under rendring.